app/src/main/java/com/sopt/now/data/remote/dto/response/reqres/Data.kt

```diff
@@ -0,0 +1,22 @@
+package com.sopt.now.data.remote.dto.response.reqres
+
+import com.sopt.now.model.Profile
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Data(
+    val avatar: String,
+    val email: String,
+    @SerialName("first_name")
+    val first_name: String,
+    val id: Int,
+    @SerialName("last_name")
+    val last_name: String,
```

Comment:
안드로이드에서는 변수명을 카멜 케이스로 작성해주시는 것을 권장합니다!
서버에서 다르게 오는 경우를 대응해주기 위해서 @SerialName 어노테이션을 활용하는 건데요, 카멜케이스 형태의 변수에 저장하는 것을 추천드려요 ~~

```suggestion
    @SerialName("last_name")
    val lastName: String,
```

Reply:
귀차나서 확장함수로 때려박았더니 이렇게 됐네요...!!
수정하겠습니당